### ISSUE TITLE
opposite sign of Mulliken charges GFN1 and GFN2

### ISSUE THREAD

**Issue by dholub689** [grimme-lab/xtb#127]
**Describe the bug**
Using GFN1 and GFN2 yields to different mulliken charge (opposite sign).

**To Reproduce**

1. happens with input (include input files)

I included a pictures with the calculated charges for anthracene, using the same xtb version having just the changed the gfn version

2. start `xtb` with (all the options here)

xtb version 6.2.2 (b9950dcfbd82755ced02c8381fbfd97c1634a289) compiled by '@Linux' on 03/03/2020

3. run `xtb` with your options and the `--verbose` flag
xtb --opt normal ant.xyz   --gfn 1
xtb --opt normal ant.xyz   --gfn 2

using: xtb --opt normal ant.xyz   --gfn 1 --input input.inp

gives also wrong charges at the output, however, it creates the xtbout.json having the charges with a flipped sign.

input.inp contains:
$chrg 0
$spin 0
$write
    json=true
$end

4. output showing the error

http://nanomatch.com/nanomatch-files/bugreports/xtbmulliken/stats.jpg

http://nanomatch.com/nanomatch-files/bugreports/xtbmulliken/pm.png

**Expected behaviour**
Both signs in GFN1 and GFN2 should be the same.

I am interested in the CM5 charges and not quite sure about their values right now. Are they calculated before or after the sign changes.

**Comment by awvwgk** [comment]
Oh, indeed there is a sign error for GFN1-xTB printout here:

https://github.com/grimme-lab/xtb/blob/b9950dcfbd82755ced02c8381fbfd97c1634a289/xtb/main_property.f90#L495

Please note, that this report was incredible hard to follow. I would really appreciate to have the necessary output and data as plain text rather than a bunch of screenshots.

**Comment by dholub689** [comment]
Thank you for fixing this so swiftly and also for your suggestion to provide all data in plaintext from now on. I will follow your advice.

Just an observation for future reference: The CM5 charges also changed significantly (for the better) with this change.
